HISTORY.rst

```diff
@@ -2,6 +2,15 @@
 Release notes
 =============
 
+3.2.0 (2021-05-17)
+------------------
+
+* Do not abstract ``requirements.in`` files any more, use them as-is (since they're intended to
+  be a spec for pip-compile_)
+
+* Next minor version of setupmeta will not auto-fill ``tests_require`` any more.
```

Comment:
nit: Should we be making behaviour changes in minor versions?

Reply:
Also, why? Was this just too magic?

Reply:
`tests_require` never worked well, and now the official doc doesn't even mention it any more -> I think it's simply going away. No one uses it really I think (most people use `tox` to run tests anyway).

setupmeta was trying to be "as complete as possible", auto-filling `tests_require` as well, as well as it could (from the "most common" ways projects state their tests reqs).

I think it's a minor change -> it's just removing a thing that never really worked anyway, and that pypa is also silently getting rid of...

Reply:
Similarly, from all the projects I surveyed, `requirements.in` is typically used with pip-compile, and all projects using it also don't let setupmeta auto-fill `install_requires`... so I think that one is a minor change as well -> it will simply be "more natural" for projects that do try to marry pip-compile + setupmeta's copy-paste reduction

Reply:
Actually `test` command (and thus related things like `tests_require` I'm guessing) was deprecated in setuptools 41.5.0: https://setuptools.readthedocs.io/en/latest/references/keywords.html

Reply:
And found official note stating `tests_require` is now deprecated :) https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#optional-dependencies